runtime/doc/change.txt

```diff
@@ -986,10 +986,10 @@ inside of strings can change!  Also see 'softtabstop' option. >
 
 							*Y*
 ["x]Y			yank [count] lines [into register x] (synonym for
-			yy, |linewise|).  If you like "Y" to work from the
-			cursor to the end of line (which is more logical,
-			but not Vi-compatible) use ":map Y y$".
-
+			yy, |linewise|). This is the builtin mapping, see
+			|Y-default| for neovim's default mapping.
```

Comment:
- the builtin normal command is not a "mapping", it is a (normal) command.
- we never say "neovim" (lowercase) in the help docs or anywhere else, and also rarely "Neovim". Almost always the help docs say "Nvim". In this case there isn't a reason to mention Nvim at all.
    - I think we need a linter for this at this point...
- There is really no reason to say anything at all here, given that the very next line mentions the default mapped nature.
    - Might seem trivial, but we need a repeatable pattern that doesn't result in clunky text all over the docs.

```suggestion
			yy, |linewise|).
```